### ISSUE TITLE
segfault when using mod_audio_stream api

### ISSUE THREAD

**Issue by ahmedggalal** [amigniter/mod_audio_stream#9]
Hello ,
I'm on Centos 7 .
running a node server that is connected to freeswitch .
when i try to use your api it fs_cli crashes and module gets unloaded .
getting this error : freeswitch kernel: freeswitch[6234]: segfault at 0 ip 00007fb0bd2dac58 sp 00007fb10ba439c0 error 4 in mod_audio_stream.so[7fb0bd2c1000+5c000 .
i tried to use other apis and all working properly .

**Comment by amigniter** [comment]
Yes, I managed to reproduce this on Centos 7.9. Issue comes from GCC compiler on Centos and a regex method which doesn't work well on Centos. Try to rebuild the module with updated changes. Regex method is replaced which will fix building and running on centos 7, and won't affect other distros.
